lib/strerror.c

```diff
@@ -453,6 +453,78 @@ curl_share_strerror(CURLSHcode error)
 #endif
 }
 
+const char *
+curl_url_strerror(CURLUcode error)
+{
+#ifndef CURL_DISABLE_VERBOSE_STRINGS
+  switch(error) {
+  case CURLUE_OK:
+    return "No error";
+
+  case CURLUE_BAD_HANDLE:
+    return "An argument that should be a CURLU pointer was passed in as a NULL";
```

Comment:
I propose:

"An invalid handle pointer was given as argument"

This way this can also be for other handle errors than NULL.

Reply:
I've synced text with #7611, but it reads slightly differently to your proposal here. Hopefully it is OK.